finalPrep/annotations.js

```diff
@@ -3,40 +3,94 @@
 
 //Build a Bear
 function buildABear(name, age, fur, clothes, specialPower) {
+  //the function buildABear is declared with 5 parameters
   var greeting = `Hey partner! My name is ${name} - will you be my friend?!`;
+  //a variable greetings is assigned a value of a string including the argument for name using interpolation
   var demographics = [name, age];
+  //a variable demographics is assigned to an array that takes the arguments for the name and age parameters
   var powerSaying = "Did you know that I can " + specialPower + " ?";
+  //a variable powerSaying is assigned a value of a string including the argument for specialPower using concatenation
   var builtBear = {
+  //a variable builtBear is assigned to an object containing 6 key-value pairs
     basicInfo: demographics,
+  //the basicInfo key is assigned the value demographics
     clothes: clothes,
+  //the clothes key is assigned the value clothes, which is a parameter for buildABear
     exterior: fur,
+  //the exterior key is assigned the value fur, which is a parameter for buildABear
     cost: 49.99,
+  //the cost key is assigned the static value of an integer with a float of 49.99
     sayings: [greeting, powerSaying, "Goodnight my friend!"],
-    isCuddly: true,
+  //the sayings is assigned an array which takes 2 variables previously assigned to strings and the static string "Goodnight my friend!"
+    isCuddly: true
```

Comment:
Very nice job in this portion of using key/value vocab, and paying attention to where your values come from.